pyproject.toml

```diff
@@ -37,52 +37,52 @@ dynamic = ["description", "version"]
 dependencies = [
   "cftime",
   # cf-xarray is differently named on conda-forge
-  "cf-xarray[all]<0.8.5; python_version == '3.8'",
-  "cf-xarray[all]; python_version >= '3.9'",
-  "climpred>=2.4.0",
+  "cf-xarray[all]",
+  "climpred >=2.4.0",
   "dask",
   "haversine",
   "h5netcdf",
   "matplotlib",
   "netCDF4",
-  "numpy<1.25",
-  "owslib>=0.24.1,<0.29", # see: https://github.com/geopython/OWSLib/issues/871
-  "pandas<2.0.0; python_version == '3.8'",
-  "pandas<2.2.0; python_version >= '3.9'",
-  "pint>=0.20",
+  "numpy",
+  "owslib >=0.24.1,<0.29", # see: https://github.com/geopython/OWSLib/issues/871
+  "pandas >=2.2.0",
+  "pint >=0.20",
   "platformdirs",
-  "pydantic>=2.0",
+  "pydantic >=2.0",
   "pymbolic",
-  "raven-hydro==0.2.4",
+  "raven-hydro ==0.2.4",
```

Comment:
Shouldn't it be 0.3.0 ?

Reply:
Precisely. It will be once we release it on PyPI and conda-forge.